server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java

```diff
@@ -8,15 +8,8 @@
 
 package org.elasticsearch.monitor.os;
 
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.anyOf;
```

Comment:
These import changes are not needed for this change,  right?

Reply:
The commit that muted these tests (https://github.com/elastic/elasticsearch/commit/09f823b1dbd64cb9b94acda7f61005a59e7aa897) moved the import statements out of the standard ordering that we use, so I just restored the original standard ordering as part of the un-mute.